developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -132,6 +132,27 @@ All rake tasks are available with the following command:
 bundle exec rake -T
 ....
 
+=== Running tests in Foreman
+Make sure to run tests from the Foreman directory.
+
+In order to run rubocop test, use the following command:
+[source, shell]
+....
+bundle exec rubocop [<path_to_file>]
+....
+
+To run Foreman's tests:
+[source, shell]
+....
+bundle exec rake test [TEST=<path_to_file>]
+....
+
+To run a specific test:
+[source, shell]
+....
+bundle exec rails test <path_to_file>:<test_line_number>
```

Comment:
This uses `rails test` vs `rake test`. Is that intentional?

Reply:
Yes, because with rake this command doesn't work.
Do you think I should just skip the last one? (with rails)